pkg/operator/ceph/disruption/clusterdisruption/osd.go

```diff
@@ -168,18 +169,33 @@ func (r *ReconcileClusterDisruption) reconcilePDBsForOSDs(
 		}
 		return fmt.Errorf("could not check cluster health: %+v", err)
 	}
+	if len(pdbStateMap.Data) < 1 {
+		pdbStateMap.Data = make(map[string]string)
+	}
 	_, ok := pdbStateMap.Data[disabledPDBKey]
 	if !ok {
 		pdbStateMap.Data[disabledPDBKey] = ""
 	}
-	if len(drainingFailureDomains) != 0 {
+	var disabledPDBTime time.Time
+	timeString, disabledPDBTimeSet := pdbStateMap.Data[disabledPDBTimeKey]
+	if disabledPDBTimeSet {
+		disabledPDBTime, err = time.Parse(time.RFC3339, timeString)
```

Comment:
we also need to handle this error right?

Reply:
Yup, nice catch!